typescript/infra/config/environments/testnet4/chains.ts

```diff
@@ -16,27 +16,11 @@ export const chainMetadataOverrides: ChainMap<Partial<ChainMetadata>> = {
   bsctestnet: {
     transactionOverrides: {
       gasPrice: 1 * 10 ** 9, // 1 gwei
-      gasPriceCap: 100 * 10 ** 9, // 100 gwei cap
     },
   },
-  arbitrumsepolia: {
+  kyvetestnet: {
     transactionOverrides: {
-      gasPriceCap: 100 * 10 ** 9, // 100 gwei cap
```

Comment:
i assume intentionally removed as there's no strong need?

Reply:
that is also correct